[BLOG] Add quansight-at-scipy2019 (#550)
Quansight/Quansight-website: apps/labs/posts/quansight-at-scipy2019.md

---

> title: 'Quansight presence at SciPy'19'

Author: ```suggestion
title: "Quansight presence at SciPy'19"
```

published: July 15, 2019

> author: Ralf Gommers

Author: ```suggestion
author: ralf-gommers
```

> description: 'This post is just a quick update to summarize Quansight's presence and contributions, as well as some of the more interesting things I noticed.'

Author: ```suggestion
description: "This post is just a quick update to summarize Quansight's presence and contributions, as well as some of the more interesting things I noticed."
```

> category: [Community, PyData ecosystem]

Author: ```suggestion
category: [Community, PyData Ecosystem]
```

Author: Update category

>   src: /posts/quansight-at-scipy2019/blog_feature_org.svg

Author: ```suggestion
featuredImage:
  src: /posts/quansight-at-scipy2019/blog_feature_org.svg
```
